tools/environment/scripts/infra/modules/selenium/run_selenium_grid.sh

```
@@ -40,8 +40,11 @@ if [[ "${CI:-}" == "true" ]]; then
   rm -f "$tmp_config"
   export SELENIUM_GRID_MOUNT
 else
-  export SELENIUM_GRID_MOUNT="$(pwd)/tools/environment/assets/selenium-grid"
-  render_config "$CONFIG_PATH" "${SELENIUM_GRID_MOUNT}/config.toml"
+  export SELENIUM_GRID_MOUNT="$(pwd)/tools/environment/assets/selenium-grid/generated"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)